### DISCUSSION TITLE
Feature request: support ability to write more readable decimal number literals

### DISCUSSION THREAD

**Discussion by scorpion9979** [PaulRBerg/evm-bn#6] [Ideas]
E.g., `fp("10,000,000.256")` is a lot easier to read than `fp("10000000.256")`.

**Comment by PaulRBerg** [comment]
Interesting. The downsides though are that (1) implementing this is not easy and (2) might lead some users to confusion. Some countries use the comma to represent the fractional part and others use the dot.
